plugins/nf-nomad/src/main/nextflow/nomad/config/NomadSecretOpts.groovy

```diff
@@ -0,0 +1,13 @@
+package nextflow.nomad.config
+
+class NomadSecretOpts {
+
+    final Boolean enable
+    final String path
+
+    NomadSecretOpts(Map map){
+        this.enable = map.containsKey('enable') ? map.get('enable') as boolean : false
+        this.path = map.path ?: "secrets/nf-nomad"
+    }
```

Comment:
@jagedn , I've renamed the `enable` to `enabled` in order to comply with the general pattern of enabling scopes as per [Nextflow configuration](https://www.nextflow.io/docs/latest/config.html#scope-k8s)